tests/acceptance/pageObjects/FilesPageElement/appSideBar.js

```diff
@@ -39,7 +39,16 @@ module.exports = {
       )
       return isVisible
     },
-    isSideBarOpenForResource: async function (resource, elementType = 'any', timeout = 500) {
+    isSideBarOpenForResource: async function (
+      resource,
+      elementType = 'any',
+      expectedToOpen = true
+    ) {
+      let timeout = 5000
+      if (!expectedToOpen) {
```

Comment:
The steps that call `isSideBarOpenForResource` all expect that the sidebar will be open.
After a quick look at the code-base, there are only steps like:
```
  /^the app-sidebar for (file|folder) "([^"]*)" should be visible on the webUI$/

'file {string} should be listed on the sidebar'
```

So, currently, nothing will call this with `expectedToOpen` set to false. But I guess it is still useful to have this parameter. That will help make people think when they add new calls this function.

Reply:
Woah, sorry for extra work then. :-( I was sure that there are occurrences where it's needed the other way around... my bad, sorry!

Reply:
No problem - it is good to code that way in general anyhow, for the future